en_US/data-integration/data-bridge-iotdb.md

```diff
@@ -0,0 +1,174 @@
+
+:::
+
+## Feature List
+
+- [Connection pool](./data-bridges.md#connection-pool)
+- [Async mode](./data-bridges.md#async-mode)
+<!-- - [Batch mode](./data-bridges.md) -->
+<!-- - [Buffer mode](./data-bridges.md) -->
+
+## Quick Start Tutorial
+
+This section introduces how to use the Apache IoTDB data bridge with a practical tutorial, covering topics like creating an Apache IoTDB instance, creating a data bridge, setting up a rule for forwarding data to the data bridge, and testing the data bridge and rule.
+
+This tutorial assumes that you have an IoTDB instance running on the local machine. If you have Apache IoTDB running remotely, please adjust the settings accordingly.
+
+### Start an Apache IoTDB Server
+
+This section introduces how to start an Apache IoTDB server using [Docker](https://www.docker.com/). Make sure to have `enable_rest_service=true` in your IoTDB's configuration.
+
+Run the following command to start an Apache IoTDB server with its REST interface enabled:
+
+```bash
+docker run -d --name iotdb-service \
+              --hostname iotdb-service \
+              -p 6667:6667 \
+              -p 18080:18080 \
+              -e enable_rest_service=true \
+              -e cn_internal_address=iotdb-service \
+              -e cn_target_config_node_list=iotdb-service:10710 \
+              -e cn_internal_port=10710 \
+              -e cn_consensus_port=10720 \
+              -e dn_rpc_address=iotdb-service \
+              -e dn_internal_address=iotdb-service \
+              -e dn_target_config_node_list=iotdb-service:10710 \
+              -e dn_mpp_data_exchange_port=10740 \
+              -e dn_schema_region_consensus_port=10750 \
+              -e dn_data_region_consensus_port=10760 \
+              -e dn_rpc_port=6667 \
+              apache/iotdb:1.1.0-standalone
+```
+
+You can find more information about running [IoTDB in Docker on Docker Hub](https://hub.docker.com/r/apache/iotdb).
+
+### Create an Apache IoTDB Data Bridge
+
+This section introduces how to create an EMQX data bridge to Apache IoTDB through Dashboard.
+
+1. Go to the Dashboard, and click **Data Integration** -> **Data Bridge** from the left navigation menu.
+
+2. Click **Create** on the top right corner of the page.
+
+3. In the **Create Data Bridge** page, click to select **Apache IoTDB**, and then click **Next**.
+
+4. Input a name for the data bridge. The name should be a combination of upper/lower case letters and numbers.
+
+5. Input the connection information:
+   * **Base URL**: Input `http//localhost:18080`, or the actual hostname/IP if the IoTDB server is running remotely.
+   * **Username**: Input the IoTDB username; The default value is `root`.
+   * **Password**: Input the IoTDB password; The default value is `root`.
+   
+6. Configure the **Payload Template**: The default value is an empty string, meaning the message payload will be forwarded as JSON-formatted text to IoTDB without modification.
```

Comment:
The Dashboard UI does not seem to have a Payload Template field. Not sure if this is a bug in the UI or the documenetation.

Reply:
I have already informed Shifan Yu about this. It was a bug and I think they are working on it now.